src/farcasterd/runtime.rs

```diff
@@ -152,11 +152,12 @@ impl Stats {
     fn success_rate(&self) -> f64 {
         let Stats { success, failure } = self;
         let total = success + failure;
-        let rate = *success as f64 / (total as f64);
+        let rate = *success as f64 / (total as f64) * 100f64;
```

Comment:
if u multiple by 100 this is not a rate

Reply:
Yes, I change this

Reply:
Done in cbdf583dca7c126c8fee095ce5afe6ca3eb6cf3b